bitbots_quintic_walk/test/rostests/test_walk.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/env python3
+import time
+
+import rospy
+from bitbots_msgs.msg import JointCommand, SupportState
+from geometry_msgs.msg import Twist, Point, Pose, Quaternion
+
+from bitbots_test.mocks import MockSubscriber
+from bitbots_test.test_case import WebotsTestCase
+from nav_msgs.msg import Odometry
+
+
+class TestWalk(WebotsTestCase):
```

Comment:
You could overwrite [`setUp`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.setUp) to construct a `/cmd_vel` publisher for the test instance since most tests need one.

You could then also overwrite [`tearDown`](https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDown) and publish a zero-Twist in it to ensure that the walking command is always cleared.

This is of course personal preference because it would make the code-flow of tests less straight-forward since additional setup and teardown would be done that is then not visible in the test functions.